### ISSUE TITLE
Committed data file for two way jitter throws error when attempting to use in figure_two_way_lmm.m

### ISSUE THREAD

**Issue by dcolli23** [Campbell-Muscle-Lab/howtos_making_figures#2]
# Description

The committed data file (`repo/code/stats/two_way_lmm/data/example_data.xlsx`) throws an error when Greg and I attempt to run the demo at `repo/code/stats/two_way_lmm/figure_two_way_lmm.m`. We believe this is due to the table being read in without column names serving as variable names. We think this is due to the seemingly repeated data in the rightmost column of the table that does not have a column header name.

# Proposal 

Deleting the rightmost column in the committed data seems to fix this issue. So we propose to delete it if it is in fact redundant.

If you agree with this, then I can make the change.

**Comment by dcolli23** [comment]
@kenatcampbellmusclelab Argh, sorry. You're correct that this is an issue for howtos_making_figures. I've migrated the issue to the correct repo.

I've pulled the latest commit and still get this issue. The error message I receive is:

![Screenshot from 2020-06-12 11-03-54](https://user-images.githubusercontent.com/44296682/84517044-afa62b80-ac9c-11ea-9c8c-d2c9061bf689.png)

When I print out `d` using the currently committed data file, I get the following:

![Screenshot from 2020-06-12 11-01-42](https://user-images.githubusercontent.com/44296682/84517057-b6cd3980-ac9c-11ea-8a0d-d28b16c0af83.png)

However, by deleting the rightmost incomplete column of the data file and rerunning the demo I get the following when printing out `d` and do not receive the same error:

![Screenshot from 2020-06-12 11-02-47](https://user-images.githubusercontent.com/44296682/84517051-b339b280-ac9c-11ea-9ddd-6b0d2c7a20c3.png)

It leads me to believe that not having the column name means that MATLAB doesn't have a full list of variable names for the columns and thus just defaults to Var1, Var2, etc.
